guest/rust/examples/basics/procedural_generation/src/client.rs

```diff
@@ -0,0 +1,367 @@
+use ambient_api::client::{material, mesh, sampler, texture};
```

Comment:
Collapse all of these into one `ambient_api` import (VSC/R-A has a "merge imports" hint)